packages/experimental/MenuButton/jest.config.js

```diff
@@ -1,2 +1,2 @@
 const { configureReactNativeJest } = require('@fluentui-react-native/scripts');
-module.exports = configureReactNativeJest('android');
+module.exports = configureReactNativeJest('ios');
```

Comment:
Why are we changing this back? 
If anything, this should be either win32 or macOS because those are the only platforms where MenuButton is available. But can we do that outside of this review?

Reply:
Have changed it to ios to maintain parity across all Button tests, we can change it to win32 when we change all the other tests as well